__snapshots__/badge-renderers.spec.js

```diff
@@ -0,0 +1,655 @@
+exports['The badge renderer "flat" template badge rendering should match snapshots: message/label, no logo 1'] = `
+
+    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="90" height="20">
+    
+      <linearGradient id="s" x2="0" y2="100%">
+        <stop offset="0" stop-color="#bbb" stop-opacity=".1"/>
+        <stop offset="1" stop-opacity=".1"/>
+      </linearGradient>
+
+      <clipPath id="r">
+        <rect width="90" height="20" rx="3" fill="#fff"/>
+      </clipPath>
+
+      <g clip-path="url(#r)">
+        <rect width="45" height="20" fill="#0f0"/>
+        <rect x="45" width="45" height="20" fill="#b3e"/>
+        <rect width="90" height="20" fill="url(#s)"/>
+      </g>
+
+      <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="110">
+        
+        <text x="235" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="350">cactus</text><text x="235" y="140" transform="scale(.1)" textLength="350">cactus</text>
+        <text x="665" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="350">grown</text><text x="665" y="140" transform="scale(.1)" textLength="350">grown</text>
+      </g>
```

Comment:
I'm in 2 minds about the best way to run these tests/record the test snapshots. I've done it with `minify=false` because I figured that if something changes its easier to read the diff than if its all smashed into one line, but the flipside of this is the tests are more brittle because they're sensitive to any whitespace changes (which isn't the case with `minify=true`). Any thoughts?